qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/QaTestApplication.kt

```diff
@@ -1,14 +1,16 @@
 package com.mapbox.navigation.qa_test_app
 
 import android.app.Application
+import com.mapbox.androidauto.MapboxCarApp
+import com.mapbox.navigation.qa_test_app.car.search.MapboxCarSearchApp
 
 class QaTestApplication : Application() {
 
     override fun onCreate() {
         super.onCreate()
 
         // Uncomment when testing android-auto
-        // MapboxCarApp.setup(this)
-        // MapboxCarSearchApp.setup(this)
+        MapboxCarApp.setup(this)
+        MapboxCarSearchApp.setup(this)
```

Comment:
I guess you uncommented these lines for testing. Do we need to comment them out? 🤔

Reply:
Good catch. Yeah. Otherwise, we will get 2 voice players when testing non drop-in UI Activities.

Reply:
So basically we should update the QA app in general to use the `SharedApp` setup so that all activities (whether using Drop-In or not) can work with AA. Correct?

Reply:
Yes but only as long as we don't instantiate separate `MapboxVoiceInstructionsPlayer` in Activities or Fragments.